todo_app/app.py

```diff
@@ -1,14 +1,26 @@
-from flask import Flask
+from flask import Flask, render_template, request
 
 from todo_app.flask_config import Config
 
+from todo_app.data import session_items
+
+
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    itemList = session_items.get_items()
+    return render_template('index.html', itemList = itemList)
+
+ 
+@app.route('/items', methods=['POST'])
+def add_item():
+    title = request.form.get('title')
+    session_items.add_item(title)
+    return render_template('confirmation.html')
```

Comment:
This is fine but I believe the instructions did actually ask you to redirect to the list of items.

By which it meant `return redirect('/')` (import the redirect function from flask)